src/Arcus.Templates.AzureFunctions.Http/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM mcr.microsoft.com/azure-functions/dotnet:4 AS base
 WORKDIR /app
```

Comment:
Is dotnet:4 the correct image that we should use as runtime base image ?

Reply:
I believe that the `dotnet:4` is about the v4 version of Azure Functions, so yes.

Reply:
Will be looked over in a follow-up PR.